src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -13,6 +13,11 @@ public partial class ImportExport
         [Fact]
         public static void ExportAutoKey()
         {
+            if (PlatformDetection.IsSmallerThan462)
```

Comment:
Can you make `IsSmallerThan462` into a method instead of a property and turn this test into a `[ConditionalFact(nameof(PlatformDetection.IsSmallerThan462)]`? That way the test would be visibly skipped in the log.

Not sure what the rules around the condition method on `ConditionalFact`s are. They may need to be in the same class as the fact.